### PR TITLE
fix for text and links colliding on download page

### PR DESCRIPTION
**Pull request by KiriHoy**
Added media query to handle text and link displaying properly on download page

### CODE REVIEW

File: src/layout/download/furnace.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ const Furnace = ({ components, _ID, _body, _parseYaml, _relativeURL }) => {
 								Object.keys( MODULES )
 									.map( ( module, i ) => (
 										<li className="furnace__component" key={ i }>
-
+										<div>
```
Comment (sukhrajghuman) [inline src/layout/download/furnace.js]: do we need this div anymore?

File: src/layout/download/furnace.js [hunk 2]
```diff
@@ -82,27 +82,27 @@ const Furnace = ({ components, _ID, _body, _parseYaml, _relativeURL }) => {
...
+																MODULES[ module ].dependencies.length > 0 && 
+																<span className="furnace__component__dependencies">
+																	<DependencyList dependencies={ MODULES[ module ].dependencies } MODULES={ MODULES } />
+																</span>
+															}
+																<a className="furnace__component__documentation" href={ _relativeURL( `/components/${ MODULES[ module ].ID }`, _ID ) }>
+																	<span className="sronly">{ MODULES[ module ].name }</span>
+																	Documentation
+																</a>
+																</div>
```
Comment (sukhrajghuman) [inline src/layout/download/furnace.js]: indentation :)

File: src/sass/components/_furnace.scss [hunk 3]
```diff
@@ -25,10 +25,16 @@
 	display: block;
 	position: relative;
 	background: $AU-color-background-shade;
+	@include AU-space( height, 5.75unit);	
+
+	@include AU-media( xs ){
```
Comment (sukhrajghuman) [inline src/sass/components/_furnace.scss]:
```suggestion
	@include AU-media( xs ) {
```

File: src/sass/components/_furnace.scss [hunk 4]
```diff
@@ -25,10 +25,16 @@
...
+	@include AU-space( height, 4unit);	
+	}
 
 	&:hover {
 	 background: $AU-color-background-alt;
 	}
+
```
Comment (sukhrajghuman) [inline src/sass/components/_furnace.scss]:
```suggestion
```

File: src/sass/components/_furnace.scss [hunk 5]
```diff
@@ -48,34 +54,35 @@
...
 
 			&:hover {
 				color: $AU-color-foreground-text;
 				text-decoration:none;
 			}
+		
+
+			
+		}
+		.au-control-input__text {
```
Comment (sukhrajghuman) [inline src/sass/components/_furnace.scss]:
```suggestion

		.au-control-input__text {
```

File: src/sass/components/_furnace.scss [hunk 6]
```diff
@@ -48,34 +54,35 @@
...
-			z-index: 100;
+			@include AU-media( md ){
+				float: right;
+			}
 
 			&:hover {
 				color: $AU-color-foreground-text;
 				text-decoration:none;
 			}
+
```
Comment (sukhrajghuman) [inline src/sass/components/_furnace.scss]:
```suggestion
```

File: src/layout/download/furnace.js [hunk 7]
```diff
@@ -61,7 +61,6 @@ const Furnace = ({ components, _ID, _body, _parseYaml, _relativeURL }) => {
 								Object.keys( MODULES )
 									.map( ( module, i ) => (
 										<li className="furnace__component" key={ i }>
```
Comment (adamzerella) [inline src/layout/download/furnace.js]:
```suggestion
										<li className="furnace__component" key={ i }>

```

Comment (adamzerella) [reply src/layout/download/furnace.js]: It's a bit easier to read if we preserve newlines, makes the change diff smaller :)

File: src/layout/download/furnace.js [hunk 8]
```diff
@@ -82,27 +81,26 @@ const Furnace = ({ components, _ID, _body, _parseYaml, _relativeURL }) => {
...
+															}
+																<a className="furnace__component__documentation" href={ _relativeURL( `/components/${ MODULES[ module ].ID }`, _ID ) }>
+																	<span className="sronly">{ MODULES[ module ].name }</span>
+																	Documentation
+																</a>
+															</div>
 													</span>
 												</span>
 											</label>
-
```
Comment (adamzerella) [inline src/layout/download/furnace.js]: newline here

File: src/layout/download/furnace.js [hunk 9]
```diff
@@ -82,27 +81,26 @@ const Furnace = ({ components, _ID, _body, _parseYaml, _relativeURL }) => {
...
-												</p>
-
-												<p className="furnace__component__documentation">
-													<a href={ _relativeURL( `/components/${ MODULES[ module ].ID }`, _ID ) }>
-														<span className="sronly">{ MODULES[ module ].name }</span>
-														Documentation
-													</a>
-												</p>
 											</div>
-
```
Comment (adamzerella) [inline src/layout/download/furnace.js]: newline here

File: src/sass/components/_furnace.scss [hunk 10]
```diff
@@ -25,6 +25,11 @@
 	display: block;
 	position: relative;
 	background: $AU-color-background-shade;
+	@include AU-space( height, 5.75unit);	
+
+	@include AU-media( xs ) {
+	@include AU-space( height, 4unit);
```
Comment (adamzerella) [inline src/sass/components/_furnace.scss]:
```suggestion
		@include AU-space( height, 4unit);	
```

File: src/sass/components/_furnace.scss [hunk 11]
```diff
@@ -25,9 +25,14 @@
 	display: block;
 	position: relative;
 	background: $AU-color-background-shade;
+	@include AU-space( height, 5.75unit);
```
Comment (sukhrajghuman) [inline src/sass/components/_furnace.scss]:
```suggestion
	@include AU-space( height, 5.75unit );	
```

File: src/sass/components/_furnace.scss [hunk 12]
```diff
@@ -25,9 +25,14 @@
 	display: block;
 	position: relative;
 	background: $AU-color-background-shade;
+	@include AU-space( height, 5.75unit);	
+
+	@include AU-media( xs ) {
+		@include AU-space( height, 4unit);
```
Comment (sukhrajghuman) [inline src/sass/components/_furnace.scss]:
```suggestion
		@include AU-space( height, 4unit );
```

**Pull request merged**